### ISSUE TITLE
Data Viewer: index values missing for Series and Dataframes with named index

### ISSUE THREAD

**Issue by JulianCologne** [microsoft/vscode-jupyter#9344]
## Environment data

-   VS Code version: 1.65.2
-   Jupyter Extension version (available under the Extensions sidebar): v2022.2.1030672458
-   Python Extension version (available under the Extensions sidebar): v2022.2.1924087327
-   OS (Windows | Mac | Linux distro) and version: MacOS 12.2.1
-   Python and/or Anaconda version: Python 3.8.12
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): conda
-   Jupyter server running: Local | Remote | Local

## Expected behaviour

Index values should not be missing in the Data Viewer if the index of the Series or Dataframe is named.

## Actual behaviour

Values in the index column are missing

## Steps to reproduce:

```python
import pandas as pd

ser = pd.Series(data=[2, 5, 3])
ser.index.name = 'time'

df = pd.DataFrame(data={'col1': [1, 2], 'col2': [3, 4]})
df.index.name = 'time'
```

![BUG_DataViewer_namedIndex](https://user-images.githubusercontent.com/25177421/158020620-92f87d4f-c3e0-4390-b104-afe267a8f4b1.png)

**Comment by amunger** [comment]
thanks for the bug, looks like this has been an issue for a while - repro'd in last month's release as well

**Comment by greazer** [comment]
Related to https://github.com/microsoft/vscode-jupyter/issues/5348.

**Comment by DonJayamanne** [comment]
I have submitted a fix for #5348 (that fixes this issue as well), this can be closed as a duplicate of #5348 as Jim has pointed out.
Closing as a duplicate
